stack-cmbm/recipes/role-vagrant.rb

```diff
@@ -0,0 +1,25 @@
+# AWS instances use RDS databases, which are not available in Vagrant
+include_recipe 'ies-mysql'
+include_recipe 'ies-mysql::dev'
+
+# Pull in the production role-* recipe.
+include_recipe 'stack-cmbm::role-nginxapp'
+
+# Configure nginx and supervisor
+include_recipe 'stack-cmbm::deploy-vagrant'
+
+execute 'install gems' do
+  command 'cd /vagrant_cmbm && bundle install'
+end
+
+execute 'setup app' do
+  command 'cd /vagrant_cmbm && bundle exec rake db:setup ads:populate'
```

Comment:
Maybe relocate those two execute statements to `deploy-vagrant` - this way, you dont have to hardcode the path, and it kind of is part of the deployment?